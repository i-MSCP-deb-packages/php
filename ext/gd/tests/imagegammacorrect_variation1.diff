001+ eab8a3508c8efa7bebaba9efa17f15a1
001- 7716c0905ae08bd84b4d6cba8969a42e