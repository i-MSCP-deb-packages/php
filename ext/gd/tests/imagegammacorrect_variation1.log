
---- EXPECTED OUTPUT
7716c0905ae08bd84b4d6cba8969a42e
---- ACTUAL OUTPUT
eab8a3508c8efa7bebaba9efa17f15a1
---- FAILED
