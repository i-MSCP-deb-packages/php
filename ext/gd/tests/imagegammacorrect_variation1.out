eab8a3508c8efa7bebaba9efa17f15a1